pkg/platform/provider/baremetal/images/images.go

```diff
@@ -112,3 +113,14 @@ func ListKubernetesImageFullNamesWithVerion(version string) []string {
 
 	return items
 }
+
+func ListKubernetesNodeImageFullNamesWithVersion(version string) []string {
+	var items []string
+	for _, name := range kubeNodeImages {
+		items = append(items, containerregistry.Image{Name: name, Tag: "v" + version}.FullName())
+	}
+
+	items = append(items, components.Pause.FullName())
+
+	return items
+}
```

Comment:
consolidate ListKubernetesImageFullNamesWithVersion and ListKubernetesNodeImageFullNamesWithVersion

Reply:
consolidate done.